src/main/java/eu/europeana/iiif/web/ManifestController.java

```diff
@@ -138,8 +138,7 @@ public ManifestController(ManifestService manifestService) {
         }
 
         Object manifest;
-        if ("3".equalsIgnoreCase(iiifVersion)) {
-            manifest = manifestService.generateManifestV3(json, addFullText, fullTextApi);
+        if ("3".equalsIgnoreCase(iiifVersion)) { manifest = manifestService.generateManifestV3(json, addFullText, fullTextApi);
```

Comment:
Sonarlint warning for code style (multiple statements per line)

Reply:
Fixed